hkmc2/shared/src/main/scala/hkmc2/semantics/Term.scala

```diff
@@ -179,12 +180,23 @@ sealed trait Statement extends AutoLocated:
       s"${cls.kind} ${cls.sym.nme}${
         cls.tparams.map(_.showDbg).mkStringOr(", ", "[", "]")}${
         cls.paramsOpt.fold("")(_.map(_.showDbg).mkString("(", ", ", ")"))} ${cls.body}"
+    case mod: ModuleDef =>
```

Comment:
This case is unreachable. `ModuleDef` is a `ClassLikeDef `.

Reply:
I think I forgot to remove after rebasing, will remove this later